### PR TITLE
Search UI added.

### PR DESCRIPTION
**Pull request by m-murad**
Fixes part of #173 

In this I have added the search UI.
Please merge this.
In next PR i will implement the search functionality.

### CODE REVIEW

File: mobile/src/main/res/menu/search.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+      xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <item android:title="@string/search"
+        android:id="@+id/menu_search"
+        app:showAsAction="always|collapseActionView"
+        android:icon="@drawable/ic_search"
+        app:actionViewClass="android.support.v7.widget.SearchView"/>
+
+</menu>
```
Comment (KeenWarrior) [inline mobile/src/main/res/menu/search.xml]: Add new line at end of file

File: mobile/src/main/res/drawable/ic_search.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+-->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+  <path
+      android:fillColor="#FFFFFFFF"
+      android:pathData="M15.5,14h-0.79l-0.28,-0.27C15.41,12.59 16,11.11 16,9.5 16,5.91 13.09,3 9.5,3S3,5.91 3,9.5 5.91,16 9.5,16c1.61,0 3.09,-0.59 4.23,-1.57l0.27,0.28v0.79l5,4.99L20.49,19l-4.99,-5zM9.5,14C7.01,14 5,11.99 5,9.5S7.01,5 9.5,5 14,7.01 14,9.5 11.99,14 9.5,14z"/>
+</vector>
```
Comment (KeenWarrior) [inline mobile/src/main/res/drawable/ic_search.xml]: add new line at end

**Comment by KeenWarrior** [comment]
@free4murad 2 minor changes plus resolving merge conflict removal needed. rest is nice :+1:

**Comment by m-murad** [comment]
@garg-anuj 
Made the changes, please review.

**Pull request closed without merging**